tensorboard/pip_package/build_pip_package.sh

```diff
@@ -153,32 +161,41 @@ smoke() {
   python -c "
 import tensorboard as tb
 assert tb.__version__ == tb.version.VERSION
-tb.summary.scalar_pb('test', 42)
 from tensorboard.plugins.projector import visualize_embeddings
-from tensorboard.plugins.beholder import Beholder, BeholderHook
 tb.notebook.start  # don't invoke; just check existence
 "
+  if [ -n "${smoke_tf}" ]; then
+    # Only test summary scalar and beholder with TF
+    python -c "
+import tensorboard as tb
+tb.summary.scalar_pb('test', 42)
+from tensorboard.plugins.beholder import Beholder, BeholderHook
+"
+  fi
 
-  # Exhaustively test various sequences of importing tf.summary.
-  test_tf_summary() {
-    # First argument is subpath to test, e.g. '' or '.compat.v2'.
-    import_attr="import tensorflow as tf; a = tf${1}.summary; a.write; a.scalar"
-    import_as="import tensorflow${1}.summary as b; b.write; b.scalar"
-    import_from="from tensorflow${1} import summary as c; c.write; c.scalar"
-    printf '%s\n' "${import_attr}" "${import_as}" "${import_from}" | python -
-    printf '%s\n' "${import_attr}" "${import_from}" "${import_as}" | python -
-    printf '%s\n' "${import_as}" "${import_attr}" "${import_from}" | python -
-    printf '%s\n' "${import_as}" "${import_from}" "${import_attr}" | python -
-    printf '%s\n' "${import_from}" "${import_attr}" "${import_as}" | python -
-    printf '%s\n' "${import_from}" "${import_as}" "${import_attr}" | python -
-  }
-  test_tf_summary '.compat.v2'
-  is_tf_2() {
-    python -c "import tensorflow as tf; assert tf.__version__[:2] == '2.'" \
-      >/dev/null 2>&1
-  }
-  if is_tf_2 ; then
-    test_tf_summary ''
+  if [ -n "${smoke_tf}" ]; then
+    # Exhaustively test various sequences of importing tf.summary
+    # but only with TF
```

Comment:
The change to this comment seems redundant to me. It’s clear both that
we _are_ only testing with TF (the immediately preceding line is the
guard) and that we _should_ only be testing with TF (it’s `tf.summary`
imports, after all). Suggest reverting.